provisioning/miniprov/hare_mp/main.py

```diff
@@ -201,6 +202,62 @@ def test(args):
         exit(-1)
 
 
+def reset(args):
```

Comment:
I think we need to cleanup installation data as well. We can remove installed things in reset `reset` option.
1. `/opt/seagate/cortx/hare` path.
2. All installations that hare component does.